javascript/new-src/.jshintrc

```diff
@@ -5,9 +5,9 @@
   "latedef": true,
   "unused": true,
   "noarg": true,
-  "indent": 1,
+  "indent": 2,
   "trailing": true,
-  "white": true,
```

Comment:
I take it with the tweaks I requested you can't leave white true globally? Can you instead just blacklist the explicit lines for now, so we can keep the majority of the codebase linted please.